src/ace.F90

```diff
@@ -378,8 +379,8 @@ subroutine read_ace_table(i_table, i_listing)
       if (data_0K) then
         continue
       else
-        call read_nu_data(nuc)
         call read_reactions(nuc)
+        call read_nu_data(nuc)
```

Comment:
What do you think about making read_ace_table an initialization routine of NuclideCE?